quarkchain/cluster/tests/test_shard_state.py

```diff
@@ -3158,3 +3158,96 @@ def test_remove_tx_from_queue_with_higher_nonce(self):
 
         state.finalize_and_add_block(b0)
         self.assertEqual(len(state.tx_queue), 0)
+
+    def test_mint_new_native_token(self):
+        id1 = Identity.create_random_identity()
+        acc1 = Address.create_from_identity(id1, full_shard_key=0)
+        env = get_test_env(genesis_account=acc1, genesis_minor_quarkash=10 ** 20)
+        state = create_default_shard_state(env=env)
+        evm_state = state.evm_state  # type: EvmState
+
+        contract_runtime_bytecode = bytes.fromhex(
```

Comment:
can refer to #792 on how to get runtime code from deployment bytecode. same for function invocation bytecode